tokio/tests/runtime_current_thread.rs

```diff
@@ -1,5 +1,4 @@
 #![warn(rust_2018_idioms)]
-#![cfg(feature = "default")]
```

Comment:
Why is this removed?

Reply:
We need better ways to organize tests & feature flags. I opted to punt to follow up work as the entire crate needs it.